apiml-security-common/build.gradle

```diff
@@ -0,0 +1,16 @@
+dependencies {
+    compile(project(':common-service-core'))
```

Comment:
Are you sure `common-service-core` is need it here, I mean it's already get in `apml-common`, so you'll lug it twice in compile?

Reply:
Good catch, removed